static/src/javascripts/projects/common/modules/navigation/sticky.js

```diff
@@ -108,7 +108,11 @@ define([
                 if (this.isTablet || this.isMobile) {
                     this.$els.navigation.removeClass('animate-down-mobile').addClass('animate-up-mobile');
                 } else {
-                    this.$els.navigation.removeClass('animate-down-desktop').addClass('animate-up-desktop');
+                    if (config.page.section === 'childrens-books-site' || config.page.shouldHideAdverts) {
+                        this.$els.navigation.css('height', '72px');
```

Comment:
What if navigation has only one row? I think we should rather check at the beginning for the navigation height and cache it and then use it from variable.